src/test/java/org/folio/edge/ncip/MainVerticleTest.java

```diff
@@ -151,41 +146,36 @@ public void testConfigCheck(TestContext context) throws Exception {
 		      .response();
 	  logger.info(resp.asPrettyString());
   }
-  
+
   @Test
   public void testErrorMessage() {
 	  ErrorMessage errorMessage = new ErrorMessage(200,"ok");
-	  assertFalse(errorMessage.equals("ok"));
+    assertEquals("ok", errorMessage.message);
   }
   
   @Test
   public void testErrorMessagesEquality() {
 	  ErrorMessage errorMessageOrig = new ErrorMessage(500,"bad request");
 	  ErrorMessage errorMessage = new ErrorMessage(200,"ok");
```

Comment:
if you formatted this file so these 2 lines with error messages also should be formatted